aurelia-slickgrid/src/aurelia-slickgrid/editors/dateEditor.ts

```diff
@@ -91,14 +91,22 @@ export class DateEditor implements Editor {
   }
 
   serializeValue() {
+    const domValue: string = this.$input.val();
```

Comment:
why not just `const domValue: string = this.$input.val() || '';` ?
That is what I've done in my other repo

Reply:
if `domValue` is anything other than a date (e.g. null, undefined or '') and you pass that to moment, the returned serialized value will be 'Invalid Date'